mithril-common/src/resource_pool.rs

```diff
@@ -211,12 +208,22 @@ impl<T: Send + Sync> Drop for ResourcePoolItem<'_, T> {
     }
 }
 
+/// Reset trait implemented by pooled resources
```

Comment:
I would have preferred to have an explanation about the meaning of the Reset trait instead of having the structure that implement it

Reply:
Done